docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Server Access Access Controls
+description: Role-based access control (RBAC) for Teleport Server Access.
+---
+
+# Server Access Role-Based Access Control
+
+Role-based access control (or RBAC, for short) allows administrators to set up
+granular access policies for Linux servers connected to Teleport.
+
+An example of a policy could be, *""server administrators have access to
+everything, QA team and engineers have full access to staging servers, and
+engineers can gain temporary access to the production server in case of
+emergency"*.
+
+For a more general description of Teleport roles and examples see [RBAC](../access-controls/introduction.mdx), as
+this section focuses on configuring RBAC for Server Access.
+
+## Role configuration
+
+Teleport's "role" resource provides the following instruments for restricting
+server access:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: developer
+spec:
+  allow:
+    # The logins array defines the OS/UNIX logins a user is allowed to use.
+    # both strings and template variables are supported in this field
+    logins: [ubuntu, debian, '{{internal.logins}}']
+
+    # node_labels: a user with this role will be allowed to connect to
+    # SSH nodes, which labels match expressions below.
+    node_labels:
+      # literal strings:
+      'env': 'test'
+      # the wildcard ('*') means "any node"
+      '*': '*'
+      # a list of alternative options:
+      'region': ['us-west-1', 'eu-central-1']
+      # regular expressions start with ^ and end with $
+      # Teleport uses golang regexp syntax.
+      # of the list example above can be expressed as:
+      'reg': '^us-west-1|eu-central-1$'
+
+    # List of host groups the created user will be added to. Any that don't already exist are created.
+    host_groups: [ubuntu, nginx, other]
+
+    # Assign the user to the sudoers group
+    host_sudoers:
+    - 'ALL=(ALL) NOPASSWD: ALL'
+
+    # Deny access to the root user
+  deny:
+    logins:
+    - root
+
+  options:
```

Comment:
This section of the page is about restricting access. Role options are covered below.

I would remove options from here to avoid introducing too many concepts at once.

Reply:
addressed